pkg/kubectl/cmd/apply.go

```diff
@@ -73,7 +73,7 @@ const (
 
 var (
 	apply_long = templates.LongDesc(i18n.T(`
-		Apply a configuration to a resource by filename or stdin.
+		Apply a configuration to the specified resource by filename or stdin with the resource name.
```

Comment:
Is it really more like : `Apply a configuration to the specified resource by filename or stdin with the exact matching resource name`  ?  I think thats a little stronger and probably more helpful in terms of helping folks avoid this issue in the future.